src/lang/en-US.ts

```diff
@@ -309,7 +309,8 @@ const EntityTable: ResolvedIntlConfig['messages'] = {
 
 const LogsDialog: ResolvedIntlConfig['messages'] = {
     'logs.default': `Waiting for logs...`,
-    'logs.toManyEmpty': `Logs for this build may have ended. Click ${CTAs['cta.restart']} to check for new logs.`,
+    'logs.tooManyEmpty': `Logs for this build may have ended. Click ${CTAs['cta.restart']} to check for new logs.`,
+    'logs.networkFailure': `We are having issues fetching logs. There may be network issues. Click ${CTAs['cta.restart']} to try again.`,
```

Comment:
>We are having issues fetching logs.

This may cause a greater amount of panic than intended. If the source of the problem falls within the user's locus of control (e.g., the stability of their network connection, clicking a button that makes a duplicate API call, refreshing the window, etc.), I would recommend writing this _error acknowledgement_ in the past tense. If the source of the problem is outside the user's locus of control, then writing the _error acknowledgement_ in the present tense communicates a proper level of severity.

>There may be network issues.

It is unclear whether the user should take action to investigate the source of the error. If a weak client connection is likely at fault, I recommend explicitly advising the user to evaluate their network connection before attempting to fetch the logs again.

```ts
`We encountered a problem retrieving logs. Please check your network connection and click ${CTAs['cta.restart']} to try again.`
```

Reply:
We cannot know the cause as not being able to reach a server could be their network connection or the server network connection.

I like that... I'll update